File: Web/re-peta/src/main/java/com/example/prenotazioni/DAO/Corso.java
package com.example.prenotazioni.DAO;

public class Corso {
    private String titolo;
    private int CFU;
    private boolean stato;

    public Corso(String titolo, int CFU, boolean stato) {
        this.titolo = titolo;
        this.CFU = CFU;
        this.stato = stato;
    }

    public static Corso defaultCorso() {
        return new Corso(null, -1, false);
    }

    public String getTitolo() {
        return titolo;
    }

    public int getCFU() {
        return CFU;
    }

    public boolean getStato() {
        return stato;
    }

    @Override
    public String toString() {
        return "{titolo = " + titolo +
                "\tCFU = " + CFU +
                " \tstato = " + stato + "}\n";
    }
}

File: App/re_peta/lib/model/corso.dart
class Corso {
  String titolo;
  int CFU;
  bool stato;

  Corso(this.titolo, this.CFU, this.stato);

  Corso.fromJson(Map<String, dynamic> json)
      : titolo = json['titolo'],
        CFU = json['CFU'],
        stato = json['stato'];

  Map<String, dynamic> toJson() =>
      {'titolo': titolo, 'CFU': CFU, 'stato': stato};

  
}